src/data.js

```diff
@@ -136,6 +136,30 @@ module.exports = [
       'SQL',
     ],
   },
+  {
+    name: 'Tuna Çağlar Gümüş',
+    description:
+      'Senior software and systems engineer. I design things and make stuff when needed.',
+    url: 'https://pikseladam.com/uses',
+    twitter: '@tc_gumus',
+    emoji: '🧿',
+    country: '🇹🇷',
+    computer: 'apple',
+    phone: 'iphone',
+    tags: [
+      'Developer',
+      'Full Stack',
+      'JavaScript',
+      'JAVA',
```

Comment:
Could you make "JAVA" -> "Java" to stay consistent with other entries - thank you!

Reply:
Got it. Thank you 👍